app/views/content_items/guide.html.erb

```diff
@@ -6,34 +6,55 @@
 <% end %>
 
 <% content_for :simple_header, true %>
+<%
+  step_navs = @content_item.content_item.parsed_content["links"]["part_of_step_navs"]
+  part_of_step_navs = false
+  part_of_step_navs = step_navs.length if step_navs
+  page_title = @content_item.title
+
+  if part_of_step_navs and @content_item.multi_page_guide?
+    page_title = @content_item.current_part_title
+  end
+%>
 
 <div class="grid-row">
   <div class="column-two-thirds">
-    <%= render 'govuk_publishing_components/components/title', { title: @content_item.title } %>
-    <% if @content_item.multi_page_guide? %>
-      <aside class="part-navigation-container" role="complementary">
-        <%= render "govuk_publishing_components/components/contents_list", aria_label: 'Pages in this guide', contents: @content_item.part_link_elements, underline_links: true %>
-      </aside>
+    <%= render 'govuk_publishing_components/components/title', { title: page_title } %>
+
+    <% if not part_of_step_navs %>
+      <% if @content_item.multi_page_guide? %>
+        <aside class="part-navigation-container" role="complementary">
+          <%= render "govuk_publishing_components/components/contents_list", aria_label: 'Pages in this guide', contents: @content_item.part_link_elements, underline_links: true %>
+        </aside>
+      <% end %>
     <% end %>
   </div>
+
   <div class="column-two-thirds">
     <% if @content_item.has_parts? %>
-      <% if @content_item.multi_page_guide? %>
-        <h1 class="part-title">
```

Comment:
It's worth noting that this is a duplicate H1 in guide pages, this is an accessibility issue that we should correct separately.